mephisto/scripts/mturk/cleanup.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+"""
+Utility script that finds, expires, and disposes HITs that may not
+have been taking down during a run that exited improperly.
+"""
+from mephisto.providers.mturk.mturk_utils import (
+    get_outstanding_hits,
+    expire_and_dispose_hits,
+)
+from mephisto.core.local_database import LocalMephistoDB
+
+db = LocalMephistoDB()
+
+all_requesters = db.find_requesters(provider_type="mturk")
+all_requesters += db.find_requesters(provider_type="mturk_sandbox")
+
+print("You have the following requesters available for mturk and mturk sandbox:")
+r_names = [r.requester_name for r in all_requesters]
+print(sorted(r_names))
+
+use_name = input("Enter the name of the requester to clear HITs from:\n>> ")
+while use_name not in r_names:
+    use_name = input(
+        f"Sorry, {use_name} is not in the requester list. "
+        f"The following are valid: {r_names}\n"
+        f"Select one:\n>> "
+    )
+
+requester = db.find_requesters(requester_name=use_name)[0]
+client = requester._get_client(requester._requester_name)
```

Comment:
just curious, what does the the `_` at the start of `_get_client` signify? does this signify a private method?

Reply:
Yeah, `_get_client` really shouldn't be being used directly in user code, as it hits on the low-level API.